### ISSUE TITLE
ZeroDivisionError

### ISSUE THREAD

**Issue by jrochkind** [how-is/how_is#91]
Just to give it a try, I gem installed and tried to run it on one of my own repos. Result, a ZeroDivisionError.  Perhaps (just wild guess) because the project has 0 open issues and open PRs? 

~~~
$ how_is jrochkind/bento_search
/Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report/html.rb:29:in `/': divided by 0 (ZeroDivisionError)
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report/html.rb:29:in `block in horizontal_bar_graph'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report/html.rb:36:in `block in horizontal_bar_graph'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report/html.rb:35:in `each'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report/html.rb:35:in `horizontal_bar_graph'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report.rb:97:in `block in <class:Report>'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report/html.rb:60:in `instance_exec'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report/html.rb:60:in `export'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report/html.rb:64:in `export_file'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is/report.rb:106:in `export_file'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/lib/how_is.rb:23:in `generate_report_file'
	from /Users/jrochkind/.gem/ruby/2.3.1/gems/how_is-11.0.0/exe/how_is:29:in `<top (required)>'
	from /Users/jrochkind/.gem/ruby/2.3.1/bin/how_is:23:in `load'
	from /Users/jrochkind/.gem/ruby/2.3.1/bin/how_is:23:in `<main>'
~~~

**Comment by duckinator** [comment]
Yeah, that's what it looks like.

Apparently last time I fixed this (#63), I forgot to add a test, then I wound up breaking it again. So I'm gonna add a test now _then_ fix it, heh.

Thanks for reporting the issue!

**Comment by duckinator** [comment]
Turns out it's not the same as #63, but it is related to their being no open issues on that repository.

**Comment by duckinator** [comment]
This'll be fixed in the next release! (v12.0)

Thanks again for reporting the issue.
